.gitignore

```diff
@@ -2,6 +2,7 @@
 BROWSE
 /build
 /build_*
+*.bak
```

Comment:
When running `RUN_FULL_CLANG_TIDY=1 ./ci/run_clang_tidy.sh` it produces `bazelrc.bak`.

Reply:
When merging #12458 this is not relevant.